pkg/aws/policies.go

```diff
@@ -813,14 +814,27 @@ func (c *awsClient) ListOperatorRoles(version string) (map[string][]Role, error)
 		if err != nil {
 			return operatorMap, err
 		}
+		skip := false
 		for _, tag := range listRoleTagsOutput.Tags {
 			switch aws.StringValue(tag.Key) {
 			case tags.ManagedPolicies:
 				if aws.StringValue(tag.Value) == tags.True {
 					operatorRole.ManagedPolicy = true
 				}
+			case tags.ClusterID:
```

Comment:
In case an operator role is missing the cluster tag, should we skip it?
Otherwise, it will be displayed among the other operator roles.

Reply:
Most operator roles will not have the tag anymore due to reusability from oidc configs. But they still should show up if you are listing all

Reply:
In case the user provides the `-c` flag, do we want to display only the operators that are associated with the cluster?

Reply:
Yes, correct

Reply:
I think in that case if we set `skip := false`, and an operator role won't have the cluster tag, we won't get into this case `tags.ClusterID` and will add the operator to the list. So it will be displayed to the user, but not associated with the cluster.

Reply:
This adds it to the map, but then https://github.com/openshift/rosa/pull/1315/files/262d693de4c85497097136a3c342884841b53826#diff-03b84a3ca830edbf2664b62deb2a277a56f599670638d5dc179467f926c92ca1R149 this retrieves only the prefix that matches the correct cluster ID

Reply:
But you are correct, if we ever want to use that function outside of list/operator-roles it wouldn't work just by itself